packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -139,8 +142,17 @@ const stakingAccountQueriesMethods = {
   getRewards: M.call().returns(VowShape),
 };
 
+/** @see {NobleMethods} */
+const nobleMethods = {
+  depositForBurn: M.call(CosmosChainAddressShape, AmountArgShape).returns(
+    VowShape,
+  ),
+};
+
 /** @see {OrchestrationAccountCommon} */
+// TODO: consider renaming to CosmosOrchAccountHolder
```

Comment:
```suggestion
// XXX: consider renaming to CosmosOrchAccountHolder
```

Curious - would changing this now have any upgrade considerations? Presumably no, since this is just an interface guard label?